### PR TITLE
added status command

### PR DESCRIPTION
**Pull request by birdonfire**
for now, just using the existing status logic

### CODE REVIEW

File: lib/werewolf.rb [hunk 1]
```diff
@@ -4,4 +4,5 @@
 require "werewolf/player"
 require "werewolf/commands/join"
 require "werewolf/commands/start"
+require "werewolf/commands/status"
```
Comment (birdonfire) [inline lib/werewolf.rb]: having to do this for every new command is gonna get old :\

File: lib/werewolf/game.rb [hunk 2]
```diff
@@ -81,6 +81,9 @@ def process_join(username, client, channel)
       end
     end
 
+    def process_status(client, channel)
+      communicate(format_status, client, channel)
+    end
```
Comment (dissemble) [inline lib/werewolf/game.rb]: you know what's good?  tests are good

Comment (birdonfire) [inline lib/werewolf/game.rb]: good catch, ty. fixt.

File: test/werewolf/game_test.rb [hunk 3]
```diff
@@ -219,6 +219,13 @@ def test_process_join_communicates_active_game_error
       game.process_join('fakeusername', 'fakeclient', 'fakechannel')
     end
 
+    def test_process_status_communicates_game_status
+      game = Game.new
+      game.stubs(:format_status).once.returns('some info about the game state')
+      game.expects(:communicate).with('some info about the game state', anything, anything)
+      game.process_status('fakeclient', 'fakechannel')
+    end
+
```
Comment (birdonfire) [inline test/werewolf/game_test.rb]: look, pa, a test

**Pull request merged**